client/X11/xf_tsmf.c

```diff
@@ -108,7 +108,11 @@ int xf_tsmf_xv_video_frame_event(TsmfClientContext* tsmf, TSMF_VIDEO_FRAME_EVENT
 		return -1001;
 
 	/* In case the player is minimized */
+#ifdef __OpenBSD__
```

Comment:
I'm totally cool with that (and would prefer it) - I just wasn't
comfortable making the call on behalf of everyone else since I am
relatively new here.

My fear is that this was hiding a bug.  Perhaps it needed to be tested to
be == 0 instead of < 0 so I wanted some more experienced eyes to take a
look at it.


Thanks,
Bryan

On Tue, Apr 28, 2015 at 3:59 PM, Hardening <notifications@github.com> wrote:

> In client/X11/xf_tsmf.c
> <https://github.com/FreeRDP/FreeRDP/pull/2583#discussion_r29282025>:
>
> > @@ -108,7 +108,11 @@ int xf_tsmf_xv_video_frame_event(TsmfClientContext* tsmf, TSMF_VIDEO_FRAME_EVENT
> >  		return -1001;
> >
> >  	/* In case the player is minimized */
> > +#ifdef __OpenBSD__
>
> @bceverly <https://github.com/bceverly> can you explain what is specific
> to OpenBSD here ?
> if the check on event->numVisibleRects is useless (which seems to be the
> case), let's just remove it for all platforms...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeRDP/FreeRDP/pull/2583/files#r29282025>.
>